pkg/ring/kv/consul/client.go

```diff
@@ -35,12 +35,12 @@ var (
 
 // Config to create a ConsulClient
 type Config struct {
-	Host              string
-	ACLToken          string
-	HTTPClientTimeout time.Duration
-	ConsistentReads   bool
-	WatchKeyRateLimit float64 // Zero disables rate limit
-	WatchKeyBurstSize int     // Burst when doing rate-limit, defaults to 1
+	Host              string        `yaml:"host"`
```

Comment:
`url`? I think it would be nice to be consistent to use `url` in the names for fields which are actually URLs.

Reply:
I agree, I think this should be consistent where relevant

Reply:
Actually, this is not a URL :-( Expected format here is host:port.